core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -287,7 +280,7 @@ export class StateGraphLayoutService {
         // depth to process now, we ignore it for now and stick it back in
         // the queue to be processed later.
         if (bestPath.indexOf(linkTarget) !== -1 &&
-            nodeData[linkTarget].depth !== nodeData[currNodeId].depth + 1) {
+          nodeData[linkTarget].depth !== nodeData[currNodeId].depth + 1) {
```

Comment:
```typescript
if (
  bestPath.indexOf(linkTarget) !== -1 &&
  nodeData[linkTarget].depth !== nodeData[currNodeId].depth + 1
) {
```

Reply:
done!
PS: did similar indentations at line no. 88 and 289 too.